index.Rmd

```diff
@@ -9,7 +9,7 @@ subtitle: Using lava flow morphology and mass wasting scars to determine if Empa
   
   The Galapagos Spreading Center is a uniquely complex setting where the Nazca and Cocos plates meet within 100km of the Galapagos Hotspot. This study will look at the seamount Emapanada(2º6’N 92ºW) which is located on the Galapagos Spreading Center, off the coast of Ecuador. I hypothesis that Empanada experienced mass-wasting concurrent to its construction rather than exclusively after it. To test this hypothesis, I will model the emplacement of each seamount based on its observed and mapped lava morphologies, and changes in flank slope. The assessment will include maps of each seamount’s morphology based on previously collected dive video/pictures.
 
-# Materials and methods
+# Materials and methods 
```

Comment:
Awesome job! It would be better if you could add details about your data, how to get data or how to collect data. 
It would let general readers to better understand the project.